jabgui/src/main/java/org/jabref/gui/groups/GroupsPreferences.java

```diff
@@ -44,6 +44,17 @@ public GroupsPreferences(boolean viewModeIntersection,
         }
     }
 
+    public GroupsPreferences() {
+        this(
+                false,
+                false,
+                false,
+                false,
+                false,
+                GroupHierarchyType.INDEPENDENT
```

Comment:
Where are the comments?

OK, maybe, you did not see, but I really wonder 

Screenshot from the issue decsription

<img width="669" height="399" alt="image" src="https://github.com/user-attachments/assets/fd462b48-1b79-497e-80fd-ee66975639f3" />